dist/COVID-VACCINE-TRIAL-schema.json

```diff
@@ -0,0 +1,330 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Covid Vaccine Trial",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "email": {
```

Comment:
should this be https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L604 ?

Reply:
@mchelen-gov the references are at bottom of file.  email, fullName and date where used from common defs. There may be others that can be used now that I am more familiar with how the schema build process works.

Reply:
@rianfowler This is what I am doing, unless I am not understanding correctly.  The common definitions used are linked in at the bottom of `schema.js` and added to the generated .json definitions section. Please let me know if I am not doing that correctly!